packages/visage/src/components/Chip.tsx

```diff
@@ -10,16 +10,76 @@ import React, {
 import { createComponent } from '../core';
 import { CloseButton } from './CloseButton';
 import { createSurfaceFocusShadow } from './shared';
+import { booleanVariant, variant } from '../variants';
 
-const ChipBase = createComponent('div', {
-  displayName: 'Chip',
-  styles: {
-    border: 1,
+const variantStyles: { [key: string]: VisageStyleSheet } = {
+  danger: {
+    backgroundColor: 'danger',
+    color: 'dangerText',
+  },
+  primary: {
+    backgroundColor: 'info',
+    color: 'infoText',
+  },
+  success: {
+    backgroundColor: 'success',
+    color: 'successText',
+  },
+  warning: {
+    backgroundColor: 'warning',
+    color: 'warningText',
+  },
+  default: {
+    backgroundColor: 'neutral',
+    color: 'neutralText',
+  },
+};
+
+const outlinedVariantStyles: { [key: string]: VisageStyleSheet } = {
+  success: {
+    backgroundColor: 'transparent',
+    borderColor: 'success',
+    borderStyle: 'solid',
+    borderWidth: 1,
+    color: 'success',
+  },
+  danger: {
+    backgroundColor: 'transparent',
+    borderColor: 'danger',
+    borderStyle: 'solid',
+    borderWidth: 1,
+    color: 'danger',
+  },
+  warning: {
+    backgroundColor: 'transparent',
+    borderColor: 'warning',
+    borderStyle: 'solid',
+    borderWidth: 1,
+    color: 'warning',
+  },
+  primary: {
+    backgroundColor: 'transparent',
     borderColor: 'primary',
     borderStyle: 'solid',
-    borderRadius: 'controlBorderRadius',
+    borderWidth: 1,
+    color: 'primary',
+  },
+  default: {
+    backgroundColor: 'transparent',
+    borderColor: 'lightAccent',
+    borderStyle: 'solid',
+    borderWidth: 1,
+    color: 'lightShadesText',
```

Comment:
we don't have `lightAccent` and `lightShadesText` colors anymore, please replace with `accent` and `shadesText`